test/db/archos/linux-x64/dbg_maps

```diff
@@ -46,7 +46,7 @@ EOF
 EXPECT=<<EOF
 9
 0
-18
+17
```

Comment:
What has changed here?

Reply:
It seems to be a problem with the system I'm using. Because that's what happens when I use the dev branch rizin. 

I will revert this change.

```
0x08048000 - 0x08049000 * usr     4K s r-x /home/aya/Source/rizinorg/rizin_dev/test/bins/elf/analysis/x86-helloworld-gcc /home/aya/Source/rizinorg/rizin_dev/test/bins/elf/analysis/x86-helloworld-gcc ; home_aya_Source_rizinorg_rizin_dev_test_bins_elf_analysis_x86_helloworld_gcc.r_x
0x08049000 - 0x0804a000 - usr     4K s rw- /home/aya/Source/rizinorg/rizin_dev/test/bins/elf/analysis/x86-helloworld-gcc /home/aya/Source/rizinorg/rizin_dev/test/bins/elf/analysis/x86-helloworld
-gcc ; home_aya_Source_rizinorg_rizin_dev_test_bins_elf_analysis_x86_helloworld_gcc.rw
0xf7c00000 - 0xf7c1e000 - usr   120K s r-- /usr/lib32/libc.so.6 /usr/lib32/libc.so.6
0xf7c1e000 - 0xf7da2000 - usr   1.5M s r-x /usr/lib32/libc.so.6 /usr/lib32/libc.so.6
0xf7da2000 - 0xf7e21000 - usr   508K s r-- /usr/lib32/libc.so.6 /usr/lib32/libc.so.6
0xf7e21000 - 0xf7e23000 - usr     8K s r-- /usr/lib32/libc.so.6 /usr/lib32/libc.so.6
0xf7e23000 - 0xf7e24000 - usr     4K s rw- /usr/lib32/libc.so.6 /usr/lib32/libc.so.6
0xf7e24000 - 0xf7e2e000 - usr    40K s rw- unk0 unk0
0xf7ee0000 - 0xf7ee2000 - usr     8K s rw- unk1 unk1
0xf7ee2000 - 0xf7ee6000 - usr    16K s r-- [vvar] [vvar] ; vvar_.r
0xf7ee6000 - 0xf7ee9000 - usr    12K s r-x [vdso] [vdso] ; vdso_.r_x
0xf7ee9000 - 0xf7eea000 - usr     4K s r-- /usr/lib32/ld-linux.so.2 /usr/lib32/ld-linux.so.2 ; usr_lib32_ld_linux.so.2.r
0xf7eea000 - 0xf7f0d000 - usr   140K s r-x /usr/lib32/ld-linux.so.2 /usr/lib32/ld-linux.so.2 ; usr_lib32_ld_linux.so.2.r_x
0xf7f0d000 - 0xf7f1b000 - usr    56K s r-- /usr/lib32/ld-linux.so.2 /usr/lib32/ld-linux.so.2 ; usr_lib32_ld_linux.so.2.r.0
0xf7f1b000 - 0xf7f1d000 - usr     8K s r-- /usr/lib32/ld-linux.so.2 /usr/lib32/ld-linux.so.2 ; usr_lib32_ld_linux.so.2.rw
0xf7f1d000 - 0xf7f1e000 - usr     4K s rw- /usr/lib32/ld-linux.so.2 /usr/lib32/ld-linux.so.2
0xffe84000 - 0xffea6000 - usr   136K s rw- [stack] [stack] ; stack_.rw
```